ampadmin.yml

```diff
@@ -0,0 +1,49 @@
+version: "3"
+
+services:
+
+    server:
+        image: ${imageAmp}
+        command: ["adm-server"]
+        deploy:
+            mode: replicated
+            replicas: 1
+            labels:
+                io.amp.role: "infrastructure"
+                io.amp.mapping: "ampadm:31315:tcp:8081"
```

Comment:
the external port to reach amplifier is 8080, the one of ampadm is 8081, They are haproxy ports in fact.
the respective port 50101 and 31315  for amplifier and ampadm are internal only and not published anymore.